File: frontend/public/newsdata/jsonScript.py
import json

category = "general"

f = open('demo.json')
  
# returns JSON object as 
# a dictionary
data = json.load(f)

new_data = []
for i in data:
    temp_dict = {}
    temp_dict["title"] = i["title"]
    temp_dict["description"] = i["description"]
    temp_dict["newsUrl"] = i["url"]
    temp_dict["imageUrl"] = i["urlToImage"]
    temp_dict["source"] = i["source"]["name"]
    temp_dict["publishedAt"] = i["publishedAt"]
    temp_dict["category"] = category

    new_data.append(temp_dict)

with open("experimentalimports.json", "w") as json_file2:
    json.dump(new_data, json_file2)    

File: frontend/public/newsdata/jsonScript.js
const input = require('./demo.json')
const fs = require('fs')
const category = "general"

let data = []
input.forEach((item)=>{
    let tmp = {}
    tmp.title = item.title
    tmp.description = item.description
    tmp.newsUrl = item.url
    tmp.imageUrl = item.urlToImage
    tmp.source = item.source.name
    tmp.publishedAt = item.publishedAt
    tmp.category = category

    data.push(tmp)
})

const str = JSON.stringify(data)

fs.writeFile('experimentalimports.json', str, (err) => {
    if (err) throw err
})